### ISSUE TITLE
Issue in pandas example

### ISSUE THREAD

**Issue by springcoil** [HypothesisWorks/hypothesis-python#866]
When I try to replicate one of the examples.
I get the following.  cc #785 

```In [20]: data_frames(
    ...: ...     columns=column(["lo", "hi"], dtype=float),
    ...: ...     rows=st.tuples(st.floats(allow_nan=False),
    ...: ...                    st.floats(allow_nan=False)).map(sorted)
    ...: ... ).example()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/strategies.py in try_convert(typ, value, name)
   1684     try:
-> 1685         return typ(value)
   1686     except TypeError:

TypeError: 'column' object is not iterable

During handling of the above exception, another exception occurred:

InvalidArgument                           Traceback (most recent call last)
<ipython-input-20-c076347d2b68> in <module>()
      2     columns=column(["lo", "hi"], dtype=float),
      3     rows=st.tuples(st.floats(allow_nan=False),
----> 4                    st.floats(allow_nan=False)).map(sorted)
      5 ).example()

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/searchstrategy/strategies.py in example(self, random)
    179                     max_iterations=1000,
    180                     database=None,
--> 181                     verbosity=Verbosity.quiet,
    182                 )
    183             )

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/core.py in find(specifier, condition, settings, random, database_key)
    836                 specifier, type(specifier).__name__
    837             ))
--> 838     specifier.validate()
    839
    840     search = specifier

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/searchstrategy/strategies.py in validate(self)
    250         try:
    251             self.validate_called = True
--> 252             self.do_validate()
    253         except:
    254             self.validate_called = False

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/searchstrategy/lazy.py in do_validate(self)
    120
    121     def do_validate(self):
--> 122         w = self.wrapped_strategy
    123         assert isinstance(w, SearchStrategy), \
    124             '%r returned non-strategy %r' % (self, w)

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/searchstrategy/lazy.py in wrapped_strategy(self)
    102
    103             base = self.__function(
--> 104                 *self.__args, **self.__kwargs
    105             )
    106             if (

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/extra/pandas/impl.py in data_frames(columns, rows, index)
    466
    467     assert columns is not None
--> 468     columns = st.try_convert(tuple, columns, 'columns')
    469
    470     rewritten_columns = []

/Users/peadarcoyle/anaconda/envs/py35/lib/python3.5/site-packages/hypothesis/strategies.py in try_convert(typ, value, name)
   1687         raise InvalidArgument(
   1688             'Cannot convert %s=%r of type %s to type %s' % (
-> 1689                 name, value, type(value).__name__, typ.__name__
   1690             )
   1691         )

InvalidArgument: Cannot convert columns=column(name=['lo', 'hi'], elements=None, dtype=<class 'float'>, fill=None, unique=False) of type column to type tuple```

**Comment by DRMacIver** [comment]
Sorry, that's a typo in the example (which is still a bug!). It should be `columns=columns(...)`, not `columns=column(...)` (there's an s missing).

**Comment by Zac-HD** [comment]
Also related to #711, which would have caught this in CI.

**Comment by Zac-HD** [comment]
Closed by #867.
